src/eureka/lib/readECF.py

```diff
@@ -50,14 +60,16 @@ def __init__(self, folder=None, file=None, **kwargs):
         '''
         if folder is None:
             folder = '.'+os.sep
```

Comment:
Did you use this implementation because `'.'+os.sep` isn't allowed in the default variable assignment on Line 33?